### PR TITLE
Shock Touch is less-than-lethal, doing less lethal damage but now doing some stamina damage, staggers instead of chaining

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Shock Touch does slightly less burn damage from its shock (5), but does a reasonable amount of stamina damage in return (20). The cooldown is also lower. This makes it a less-than-lethal attack. The damage is reduced by the targets energy armor. Obviously, still does nothing against targets immune to shocks.

If it has a power chromosome, rather than chain, it instead staggers for 6 seconds.

Still does full lethal damage to any non-carbon mobs.

## Why It's Good For The Game

Shock touch is vaguely useful, but fairly mediocre except as a surprise. While it does indeed remove weapons from peoples hands, just disarming mostly for free isn't quite enough to turn a fight sometimes. I've definitely disarmed someone only for them to pull out a new weapon and keep fighting, since I didn't stop them actually responding easily. The confusion effect is fairly light, and it wasn't quite working as a 'zap and run' kind of power.

With this change, it definitely makes it more useful for defensive purposes all around, without necessarily being too good as an actual fight closer. An emergency tool at most, and maybe a decent attack to mix in during the right circumstances. 

Especially, much, much more enticing to security, who I think are the ones this power should be most attractive too but I've struggled to convince sec players to take the power when offered (they're quite happy to take insulated though).

It also makes the cooldown reduction or power increase an actual choice. Attack more often, and reduce stamina faster, or get a stagger off and potentially slow a target down and make them easier to hit/easier for you or harder for them to flee?

The power upgrade, previously, was complete dogshit. It mostly was just for friendly firing a bunch of people in the vicinity of the target for fairly minor damage at best, and didn't really contribute very much to ending a fight sooner. Meanwhile, spamming more shock touches means more damage output, so it was always the best option to take.

## Changelog
:cl:
balance: Shock touch is now less-than-lethal, dealing a decent amount of stamina damage at the cost of lethal damage. Lower cooldown. Power chromosome makes it force a stagger rather than cause a chain lightning effect (this used to do like only 5 damage so...)
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
does this keep "chaining" for people pulling eachother? it's a pretty cool sec conga counter and it's a shame to see that go

**Comment by necromanceranne** [comment]
> I don't see the point of nerfing the damage but I like other changes

Its the pattern of 'less-than-lethal' to have quite low lethal damage, but comparably decent nonlethal. If it was 20 stamina and 15 lethal, you still have a lethal weapon but it also happens to be pushing them into stamina crit faster than nonlethal. Just compare the values stuff like rubber shot and the like.

**Comment by necromanceranne** [comment]
> does this keep "chaining" for people pulling eachother? it's a pretty cool sec conga counter and it's a shame to see that go

Different mechanics. I think thats just a consequence of ``electrocute_act`` which this still does.

**Comment by Xkallubar** [comment]
No more tesla zap? Guh

**Comment by moocowswag** [comment]
The tesla zap upgrade was meant to be used with shocktouch's prerequisite (insulation) and can be used to punish conga lines (a rev shock touching a conga line of sec would do 12+ armor ignoring damage to every person in the line due to the way tesla shocks double dips on people that are connected via grabs because each person gets individually shocked then those shocks spread to everyone else for more and more damage to every single individual)

that 12+ damage is 1 or even 2 less hits for every person to go into crit (because again, armor exists and this ignores that)

Basically, it's not something you instantly win a fight with, it's an opener.

It's not massively satisfying to do solo but you can give a bunch of people these 2 mutations to spam chain lightening in a group fight which is the only way you can balance a mutation that can be shared very easily.

**Comment by necromanceranne** [comment]
It's about 12+, this may have only been doing the shock chain once from the initial zap, which instantly ends pulling, and not from two shock chains from the next ``electrocute_act`` from the tesla zap. if it was working like that, that's closer to a triple dip and likely would do way more damage. I'll have to test it but I suspect that this interaction is still entirely possible.

**Comment by moocowswag** [comment]
> It's about 12+, this may have only been doing the shock chain once from the initial zap, which instantly ends pulling, and not from two shock chains from the next `electrocute_act` from the tesla zap. if it was working like that, that's closer to a triple dip and likely would do way more damage. I'll have to test it but I suspect that this interaction is still entirely possible.

I think shocks either through the electrocute act or the zap itself loses some of its initial damage as it travels, either this is a flat reduction or % or none at all because im not sure.

**Comment by necromanceranne** [comment]
If so long as the mechanic is still possible, I think its probably not significant an issue for it to do less damage overall for the sake of making it more generally useful outside of a specific niche.

**Comment by Eronymun** [comment]
Oh cool, stun hand.

Wish it did just a little more Stam damage, like 25.
